### PR TITLE
Fix #3725: Todo looks different from note in LaTeX output

### LINKED ISSUE #3725: Todo looks different from note in LaTeX output
### Problem
For HTML output, the `todo` directive looks the same as a `note` with a different title. This is OK:
todo-html

For LaTeX output, the contents of the `todo` directive is placed on a new line instead of next to the title and is missing the colon:
todo-latex

The generated LaTeX code is:
``` latex
\begin{sphinxadmonition}{warning}{Warning:}
This is a warning
\end{sphinxadmonition}

\begin{sphinxadmonition}{note}{Note:}
This is a note
\end{sphinxadmonition}

\begin{sphinxadmonition}{note}{Todo}

This is a todo
\end{sphinxadmonition}
```

#### Procedure to reproduce the problem

Use the following rst input and build HTML and PDF:

``` rst
.. warning:: This is a warning

.. note:: This is a note

.. todo:: This is a todo
```

#### Expected results
I expect the `todo` directive to have the same output as a `note`. The generated LaTeX should be:
``` latex
\begin{sphinxadmonition}{note}{Todo:}
This is a todo
\end{sphinxadmonition}
```

This means removing the empty line which causes the content to become a new paragraph and adding the colon in the title.

### Environment info
- OS: OSX 10.10.5
- Python version: 2.7.13
- Sphinx version: 1.5.5
- docutils 0.13.1
- TeX Live 2016

### PR DESCRIPTION
**Pull request by tk0miya**
refs: #3725

### CODE REVIEW

File: sphinx/ext/todo.py [hunk 1]
```diff
@@ -209,6 +210,17 @@ def depart_todo_node(self, node):
     self.depart_admonition(node)
 
 
+def latex_visit_todo_node(self, node):
+    # type: (nodes.NodeVisitor, todo_node) -> None
+    title = node.pop(0).astext().translate(tex_escape_map)
+    self.body.append('\n\\begin{sphinxadmonition}{note}{%s:}' % title)
```
Comment (jfbu) [inline sphinx/ext/todo.py]: No ``u`` needed for Python 2 ?

**Review by jfbu — COMMENTED** [review]
LGTM, see question about ``u``. (I am a no-ob in Python2, well in Python in general).

File: sphinx/ext/todo.py (hunk 1 above)
Comment (tk0miya) [reply sphinx/ext/todo.py]: Good point. Indeed, we usually use `u` suffix to represent strings. I will update this.

Note: In py2, the strings contain only ASCII chars are treated as both `str` and `unicode` objects. So it is not needed to use `u` suffix to represent them (we have to use it if the string contains non-ASCII chars). But I think using the suffix always is simple and confuse-less.

Comment (tk0miya) [reply sphinx/ext/todo.py]: updated

Comment (jfbu) [reply sphinx/ext/todo.py]: I guess as potential Unicode chars all come from variable `title` they may be already ok even without `u` ? (this just indicates how ignorant I am)

Comment (tk0miya) [reply sphinx/ext/todo.py]: py2 interpreter converts `str` object to `unicode` automatically if `title` is unicode object::
```
% python2
>>> type("str %s" % "str")
<type 'str'>
>>> type("str %s" % u"unicode")
<type 'unicode'>
```
So either is okay in this case.

**Review by jfbu — APPROVED** [review]
LGTM !

File: sphinx/ext/todo.py (hunk 1 above)
Comment (jfbu) [reply sphinx/ext/todo.py]: thanks for being very good and patient Python teacher ;-)

**Comment by tk0miya** [comment]
Thank you for quick reviewing :-)

**Pull request merged**